e2e-jar-test/core/src/main/java/net/snowflake/IngestTestUtils.java

```diff
@@ -24,6 +22,8 @@
 import java.util.Map;
 import java.util.Properties;
 import java.util.UUID;
+import net.snowflake.client.jdbc.internal.fasterxml.jackson.databind.JsonNode;
```

Comment:
Since we are using JDBC's method [here](https://github.com/snowflakedb/snowflake-ingest-java/blob/000a13912c1827c34365cbc8148fef744a943fee/src/main/java/net/snowflake/ingest/streaming/internal/StreamingIngestStage.java#L276) after adding `ConfigResponse`. Aligning Jackson version is needed.

Reply:
1. can we remove com.fasterxml as a direct dependency of our SDK's pom.xml? I don't want future PR authors to accidentally pull in com.fasterxml directly.
2. A big chunk of changes in the PR is now just import changes :/ Suggestion for future - you could try to split your commit history so that there's one commit with just such a blind change, and another commit with the actual logical changes being reviewed. ok how it is for now.

Reply:
Yes, sorry for the typo.

Reply:
Removed in pom.xml.

Reply:
still not convinced about this change - two different classes with the name JsonNode in completely different packages / namespaces don't really need to be the same unless we have some pieces of code using package1.JsonNode interacting with other pieces of code that expect package2.JsonNode.

I'm not sure where we have this situation, the only place where jdbc...JsonNode is used is `StreamingIngestStage.java` - there is no outgoing or incoming call that needs every other usage of JsonNode across the project to be using jdbc...JsonNode.

If this is a pure cleanup that you want to do that's not really needed for what you were trying to do in this PR, it should not be mixed in with feature changes IMHO. LMK if I misunderstood your changes in some way?

Reply:
The problem I faced was that in [makeClientConfigureCall->executeWithRetries](https://github.com/snowflakedb/snowflake-ingest-java/blob/abe040568fdb7202952859b2e02e80e5b7d9dfa7/src/main/java/net/snowflake/ingest/streaming/internal/StreamingIngestStage.java#L342), the [executeWithRetires](https://github.com/snowflakedb/snowflake-ingest-java/blob/188632ad70e1bc15de4ef076961d7069b57d8af6/src/main/java/net/snowflake/ingest/streaming/internal/StreamingIngestUtils.java#L7) call [unmarshallStreamingIngestResponse](https://github.com/snowflakedb/snowflake-ingest-java/blob/188632ad70e1bc15de4ef076961d7069b57d8af6/src/main/java/net/snowflake/ingest/connection/ServiceResponseHandler.java#L200) which use Jackson object mappers. Though there's no clear limitation that the Jackson annotation and mapper should be using the same version, I found that if we tried to use different jackson (com.fasterxml) for object mapper and (jdbc.fasterxml) for annotation in `ConfigureResponse` it'll fail. This implies that all classes using `executeWithRetires` should be using the same Jackson package, as there're all using the same mapper in `unmarshallStreamingIngestResponse`.

Since `parseClientConfigureResponse` should be returning jdbc..JsonNode, and it also requires mapper to unmarshall the response, I thought that aligning the Jackson package will make life easier. Instead facing this issue in future as there's no warning for annotation & mapper version mismatch but just returning an empty node which I spent plenty of time to figure out this issue.


We can still use String as intermediate object to convert com....JsonNode to com...JsonNode without aligning Jackson version:
```
com....Object mapper = new com....ObjectMapper();
String jsonString = mapper.writeValueAsString(configureResponse);
net.snowflake.client.jdbc....Object jdbcMapper = new net.snowflake.client.jdbc....ObjectMapper();
net.snowflake.client.jdbc....JsonNode jsonNode = jdbcMapper.readTree(jsonString);
```

What do you think?